lib/widget/evaluation_form/post-op-home/adl_form.dart

```diff
@@ -1772,6 +1775,21 @@ class _ADLFormState extends State<ADLForm> {
                         ;
                         print(totalscore);
                         result(totalscore);
+                        Map<String, dynamic> formDataToDB = {
+                          'Question1': score1,
+                          'Question2': score2,
+                          'Question3': score3,
+                          'Question4': score4,
+                          'Question5': score5,
+                          'Question6': score6,
+                          'Question7': score7,
+                          'Question8': score8,
+                          'Question9': score9,
+                          'Question10': score10,
+                          'TotalscoreADL': totalscore,
```

Comment:
Can we change the key to be more meaningful as possible  ?  @PormpimonPang

Reply:
ok